### PR TITLE
Clarifies the hotkey preference name, removes obsolete SSinput things

### PR DESCRIPTION
**Pull request by DominikPanic**
Tested locally, Psy said most of this is done via keybindings so there's no reason to keep this code around as it only makes stuff confusing.

Fixes: #2399

### CODE REVIEW

**Comment by Rohesie** [comment]
2302 in. This ready?

**Comment by DominikPanic** [comment]
> 
> 
> 2302 in. This ready?

No, it needs code changes

**Comment by DominikPanic** [comment]
Should technically be ready now though I cannot guarantee that this will work as testing this is difficult

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -998,8 +998,8 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			if (choice == "Cancel")
 				ShowKeybindings(user)
 				return
-			hotkeys = (choice == "Hotkey")
-			key_bindings = (hotkeys) ? deepCopyList(GLOB.hotkey_keybinding_list_by_key) : deepCopyList(GLOB.classic_keybinding_list_by_key)
+			focus_chat = (choice == "Hotkey")
```
Comment (psykzz) [inline code/modules/client/preferences.dm]: If we don't want keybindings to hit focus_chat, this should be removed.

Comment (DominikPanic) [reply code/modules/client/preferences.dm]: It's still preferable for it to be there, as most users of classic keybindings want to combine those.

**Comment by DominikPanic** [comment]
Should be ready

**Pull request merged**